Snippets/ASB/ASB_7/Usage.cs

```diff
@@ -78,5 +79,17 @@ void TopologySelectionUpgradeGuide(EndpointConfiguration endpointConfiguration)
 
             #endregion
         }
+
+        void AutoLockRenewal(EndpointConfiguration endpointConfiguration)
+        {
+            var maximumProcessingTime = TimeSpan.FromMinutes(5);
+
+            #region asb-auto-lock-renewal
+
+            endpointConfiguration.UseTransport<AzureServiceBusTransport>()
+                .MessageReceivers().AutoRenewTimeout(maximumProcessingTime);
```

Comment:
would it be better to put `TimeSpan.FromMinutes(5);` into this snippet to clearly illustrate that it is a timespan?